### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.1.0-beta1

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,93 @@
...
+# 8.1.0-beta1 (2021-07-22)
+
+### Features and enhancements
+
+* **Alerting:** Add alertmanager notifications tab. [#35759](https://github.com/grafana/grafana/pull/35759), [@nathanrodman](https://github.com/nathanrodman)
+* **Alerting:** add button to deactivate current alertmanager configuration. [#36951](https://github.com/grafana/grafana/pull/36951), [@domasx2](https://github.com/domasx2)
+* **Alerting:** add toggle in Loki/Prom datasource config to opt out of alerting ui. [#36552](https://github.com/grafana/grafana/pull/36552), [@domasx2](https://github.com/domasx2)
+* **Alerting:** allow  any "evaluate for" value >=0 in the alert rule form. [#35807](https://github.com/grafana/grafana/pull/35807), [@domasx2](https://github.com/domasx2)
+* **Alerting:** if cortex alertmanager returns empty user config, load default config from status endpoint. [#35769](https://github.com/grafana/grafana/pull/35769), [@domasx2](https://github.com/domasx2)
+* **Alerting:** view to display alert rule and its underlying data. [#35546](https://github.com/grafana/grafana/pull/35546), [@mckn](https://github.com/mckn)
```
Comment (osg-grafana) [inline CHANGELOG.md:13]: I do not know what this means from the title without digging. Moving on to other titles for the time being. Note that I am making changes upstream.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,93 @@
...
+* **Annotation Panel:** release annotation panel. [#36959](https://github.com/grafana/grafana/pull/36959), [@ryantxu](https://github.com/ryantxu)
```
Comment (osg-grafana) [inline CHANGELOG.md:14]: Vet my upstream change for accuracy.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,93 @@
...
+* **Annotations:** Add typeahead support for tags in builtin annotations. [#36377](https://github.com/grafana/grafana/pull/36377), [@ashharrison90](https://github.com/ashharrison90)
+* **AzureMonitor:** Add curated dashboards for Azure services. [#35356](https://github.com/grafana/grafana/pull/35356), [@avidhanju](https://github.com/avidhanju)
+* **AzureMonitor:** Add support for deep links to Azure Portal for Metrics. [#32273](https://github.com/grafana/grafana/pull/32273), [@shuotli](https://github.com/shuotli)
+* **AzureMonitor:** Curated dashboards cont. [#35959](https://github.com/grafana/grafana/pull/35959), [@shuotli](https://github.com/shuotli)
```
Comment (osg-grafana) [inline CHANGELOG.md:18]: @joshhunt, this title needs to contain stand-alone information that the reader can understand without opening the PR.

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Please vet lines in question, and note that I made some changes upstream. Jumping on a call, but in the meantime, use the following the model for titles.

Ensure all titles are in imperative form with a meaningful prefix:

`Alerting: Add Alertmanager notification tab`

File: CHANGELOG.md (hunk 3 above)
Comment (joshhunt) [reply CHANGELOG.md:18]: This PR was not actually merged - this line should be removed.

I guess the PR bot doesnt check if it was actually merged or not?

I removed the label. Will the changelog be regenerated or should I edit this manually?

File: docs/sources/release-notes/release-notes-8-1-0-beta1.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+* **AzureMonitor:** Fix queries with Microsoft.NetApp/../../volumes resources. [#32661](https://github.com/grafana/grafana/pull/32661), [@pckls](https://github.com/pckls)
+* **AzureMonitor:** Request and concat subsequent resource pages. [#36958](https://github.com/grafana/grafana/pull/36958), [@andresmgot](https://github.com/andresmgot)
+* **Bug:** Fix parse duration for day. [#36942](https://github.com/grafana/grafana/pull/36942), [@idafurjes](https://github.com/idafurjes)
+* **Datasources:** Improve error handling for error messages. [#35120](https://github.com/grafana/grafana/pull/35120), [@ifrost](https://github.com/ifrost)
+* **Explore:** Correct the functionality of shift-enter shortcut across all uses. [#36600](https://github.com/grafana/grafana/pull/36600), [@ivanahuckova](https://github.com/ivanahuckova)
+* **Explore:** Show all dataFrames in data tab in Inspector. [#32161](https://github.com/grafana/grafana/pull/32161), [@ivanahuckova](https://github.com/ivanahuckova)
+* **Explore:** change border radius for refresh picker component. [#35821](https://github.com/grafana/grafana/pull/35821), [@olbo98](https://github.com/olbo98)
```
Comment (wbrowne) [inline docs/sources/release-notes/release-notes-8-1-0-beta1.md:69]:
```suggestion
```

Bug in the changelog generation that doesn't check if PRs are merged

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,93 @@
...
+* **AzureMonitor:** Fix queries with Microsoft.NetApp/../../volumes resources. [#32661](https://github.com/grafana/grafana/pull/32661), [@pckls](https://github.com/pckls)
+* **AzureMonitor:** Request and concat subsequent resource pages. [#36958](https://github.com/grafana/grafana/pull/36958), [@andresmgot](https://github.com/andresmgot)
+* **Bug:** Fix parse duration for day. [#36942](https://github.com/grafana/grafana/pull/36942), [@idafurjes](https://github.com/idafurjes)
+* **Datasources:** Improve error handling for error messages. [#35120](https://github.com/grafana/grafana/pull/35120), [@ifrost](https://github.com/ifrost)
+* **Explore:** Correct the functionality of shift-enter shortcut across all uses. [#36600](https://github.com/grafana/grafana/pull/36600), [@ivanahuckova](https://github.com/ivanahuckova)
+* **Explore:** Show all dataFrames in data tab in Inspector. [#32161](https://github.com/grafana/grafana/pull/32161), [@ivanahuckova](https://github.com/ivanahuckova)
+* **Explore:** change border radius for refresh picker component. [#35821](https://github.com/grafana/grafana/pull/35821), [@olbo98](https://github.com/olbo98)
```
Comment (wbrowne) [inline CHANGELOG.md:64]:
```suggestion
```

Bug in the changelog generation that doesn't check if PRs are merged

File: CHANGELOG.md (hunk 3 above)
Comment (osg-grafana) [reply CHANGELOG.md:18]: They will be regenerated.

**Pull request closed without merging**